lib-api/src/main/java/org/ergoplatform/appkit/ErgoProver.java

```diff
@@ -94,22 +97,26 @@
     /**
      * Verifies a signature on given (arbitrary) message for a given public key.
      *
-     * @param sigmaTree public key (represented as a tree)
-     * @param message message to verify
+     * @param sigmaTree     public key (represented as a tree)
+     * @param message       message to verify
      * @param signedMessage signature for the message
      * @return whether signature is valid or not
      */
-    boolean verifySignature(SigmaBoolean sigmaTree, byte[] message, byte[] signedMessage);
+    static boolean verifySignature(SigmaBoolean sigmaTree, byte[] message, byte[] signedMessage) {
```

Comment:
signedMessage name is misleading, better call it `signature` as in the original method of interpreter.